### PR TITLE
[IOTDB-3825] Reinforce Cluster IT framework's startup check

### PR DESCRIPTION
**Pull request by MiniSho**
Check Nodes' states by using showCluster, and the test fails when states are not "Running".

### CODE REVIEW

File: integration-test/src/main/java/org/apache/iotdb/it/env/AbstractEnv.java [hunk 1]
```diff
@@ -185,12 +187,40 @@ public void testWorking() {
...
 
+  private void checkNodeHeartbeat() {
+    TShowClusterResp showClusterResp;
+    try (SyncConfigNodeIServiceClient client =
+        (SyncConfigNodeIServiceClient) getConfigNodeConnection()) {
+      boolean flag;
+      for (int i = 0; i < 30; i++) {
+        flag = true;
+        showClusterResp = client.showCluster();
+        Map<Integer, String> nodeStatus = showClusterResp.getNodeStatus();
```
Comment (CRZbulabula) [inline integration-test/src/main/java/org/apache/iotdb/it/env/AbstractEnv.java:207]: We should also check the size of nodeStatus map

**Review by CRZbulabula — CHANGES_REQUESTED** [review]
PTAL~

File: integration-test/src/main/java/org/apache/iotdb/it/env/AbstractEnv.java [hunk 2]
```diff
@@ -185,12 +187,40 @@ public void testWorking() {
...
+            break;
+          }
+        }
+        if (flag) {
+          return;
+        }
+        Thread.sleep(1000);
+      }
+    } catch (Exception e) {
+      fail("Not all nodes are in the Running states");
```
Comment (Beyyes) [inline integration-test/src/main/java/org/apache/iotdb/it/env/AbstractEnv.java:220]: If we need sleeping some time to wait that all the status of nodes have been running?

File: integration-test/src/main/java/org/apache/iotdb/it/env/AbstractEnv.java (hunk 1 above)
Comment (MiniSho) [reply integration-test/src/main/java/org/apache/iotdb/it/env/AbstractEnv.java:207]: done

**Review by Beyyes — APPROVED** [review]

**Review by CRZbulabula — APPROVED** [review]
LGTM~

**Pull request merged**